core/server/api/shared/pipeline.js

```diff
@@ -156,9 +159,6 @@ const pipeline = (apiController, apiUtils) => {
                 })
                 .then(() => {
                     return frame.response;
-                })
-                .catch((err) => {
-                    throw {err, docName, method};
```

Comment:
@gargol Which consequences has this change on the error message **right now**? e.g. the site uses browse & read only, which most common error is a 404.

Reply:
As long as `frameOptions` are populated with docName and method that map to these rules: https://github.com/TryGhost/Ghost/blob/master/core/server/web/shared/middlewares/error-handler.js#L96-L106 there should be no effect. The 404 for `read`/`browse` would map to `read/list {resource}` and this translation:
```
"NotFoundError": "Resource not found error, cannot {action}.",
```

Reply:
perfect 👍